### PR TITLE
XW-2874 | styling video play button with wiki styles

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-2874

SVG is inline cause we don't have it in design system yet. I'll create a ticket for extracting it to DS.

@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/ArticleVideo/styles/article-video.scss [hunk 1]
```diff
@@ -72,12 +74,27 @@ $spinnerSize: 48;
 		text-transform: uppercase;
 	}
 
-	.video-play-button {
-		bottom: 15px;
-		height: 120px;
+	.video-wiki-play-button {
+		align-items: center;
+		background-color: $color-buttons;
+		border: 6px solid rgba(0,0,0,0.08);
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/styles/article-video.scss]: spaces
`0` is not required in front of dot.

File: extensions/wikia/ArticleVideo/styles/article-video.scss [hunk 2]
```diff
@@ -72,12 +74,27 @@ $spinnerSize: 48;
...
-		right: 15px;
-		width: 120px;
+		right: 35px;
+		width: 80px;
+
+		svg {
+			margin-left: 5px;
+
+			path {
+				fill: $color-button-text;
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/styles/article-video.scss]: can we apply this rule on svg?

File: extensions/wikia/ArticleVideo/templates/ArticleVideo_index.php [hunk 3]
```diff
@@ -15,7 +15,15 @@
 							$videoDetails['time'] )->escaped(); ?></div>
 					<div class="video-title"><?= htmlspecialchars( $videoDetails['title'] ); ?></div>
 				</div>
-				<img src="<?= $videoPlayButtonUrl; ?>" class="video-play-button">
+				<div class="video-wiki-play-button">
+					<svg width="22" height="30" viewBox="0 0 22 30"
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/templates/ArticleVideo_index.php]: Please add some comment with TODO and ticket number.

Comment (rogatty) [reply extensions/wikia/ArticleVideo/templates/ArticleVideo_index.php]: Make sure it's a PHP comment :wink:

File: extensions/wikia/ArticleVideo/templates/ArticleVideo_index.php [hunk 4]
```diff
@@ -15,7 +15,15 @@
...
+					     xmlns="http://www.w3.org/2000/svg">
+						<g fill="none" fill-rule="evenodd">
```
Comment (rogatty) [inline extensions/wikia/ArticleVideo/templates/ArticleVideo_index.php]: If you remove this group, then it should be possible to do what @kvas-damian suggested in https://github.com/Wikia/app/pull/12488/files#r103205998

**Review by kvas-damian — APPROVED** [review]

**Review by rogatty — APPROVED** [review]

**Pull request merged**